src/conf.c

```diff
@@ -19,13 +19,15 @@
 
 typedef bool (*conf_item_parser)(const char *str, void *result, char **errmsg);
 typedef bool (*conf_item_verifier)(void *value, char **errmsg);
+typedef const char *(*conf_item_printer)(void *value);
```

Comment:
The convention in ccache is to name functions that convert types to string like `format_*`, so please rename this to `conf_item_formatter` and the `print_*` functions accordingly.

Reply:
Done, thanks!